### DISCUSSION TITLE
Experience on that: Optical Rain Sensor Dual-Channel Outdoor Meteorological Rain Gauge

### DISCUSSION THREAD

**Discussion by Mark-Muc** [arendst/Tasmota#21593] [Q&A] [answered]
Has someone connected such a sensor to Tasmota?
What to choose? 
- RS485
- pulse

and how to connect correctly to ESP 32 device?

https://www.aliexpress.com/item/1005006978803529.html?algo_exp_id=62a0474a-709c-4502-9c03-86517b372afa-6&pdp_npi=4%40dis%21EUR%2168.86%2168.86%21%21%21531.72%21531.72%21%40211b618e17178300488776741edea2%2112000038926340414%21sea%21DE%21750916284%21&curPageLogUid=B6MUEdm66gMY&utparam-url=scene%3Asearch%7Cquery_from%3A

Thanks for any help!
Mark_Muc

**Comment by arendst** [comment]
This one works reliably https://rainsensors.com/products/rg-15/ and is supported by tasmota.

**Reply by Mark-Muc** [reply]
OK thank you!!
I'll look for this one!
Regards
Mark_Muc

**Reply by barbudor** [reply]
Price 😭

**Reply by Mark-Muc** [reply]
1 * 99€
https://rainsensors.com/products/rg-15/
![image](https://github.com/arendst/Tasmota/assets/67410847/1093dac6-fb6b-4e8d-8d21-da551a00f27a)

**Reply by barbudor** [reply]
The 1st one you listed was already not cheap
But this one...

**Reply by Mark-Muc** [reply]
if you buy 3 (who neeed it??) it is nearly the same price.
What I like on the first device is the housing, mad of steel and it is much more "stylish".

Mark-Muc

**Reply by barbudor** [reply]
Style is everything 🤣

**Reply by sfromis** [reply, marked as answer]
I suppose that it might be feasible to experimentally try the first one, and my guess is that it may be doable, just not something which can be listed as install, config, go.

Do notice my careful sprinkling of weasel words.....
